<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionDetailsView</class>
 <widget class="QWidget" name="SelectionDetailsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="SelectionCropView" name="selectionCropView" native="true"/>
     <widget class="MaskView" name="selectionMaskView" native="true"/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkWrongSpecies">
     <property name="text">
      <string>Wrong Species</string>
     </property>
     <property name="shortcut">
      <string>F1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="speciesComboBox">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ALB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BET</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DOL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LAG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OTHER</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SHARK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>YFT</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkSmallPart">
     <property name="text">
      <string>Small Part</string>
     </property>
     <property name="shortcut">
      <string>F2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkVeryLowQuality">
     <property name="text">
      <string>Very Low Quality</string>
     </property>
     <property name="shortcut">
      <string>F3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkIgnored">
     <property name="text">
      <string>Ignored</string>
     </property>
     <property name="shortcut">
      <string>F4</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectionCropView</class>
   <extends>QWidget</extends>
   <header>selectioncropview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MaskView</class>
   <extends>QWidget</extends>
   <header>maskview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
